### PR TITLE
cyclow implementation of dropdown demo

### PR DESCRIPTION
**Pull request by ghost**
I could not build bundle.js for this demo. Can you help me, please?

### CODE REVIEW

**Comment by krawaller** [comment]
Hmm, not sure what failed for you. I pulled down your branch, and have built both the demo and the site without any errors being thrown.

**Comment by krawaller** [comment]
Interesting new (?) cyclow syntax with the `in.` and `dom.` event prefixes!

**Comment by ghost** [comment]
thank you! it's new in the last release: 0.5.0 (docs in progress...)

**Comment by ghost** [comment]
this implementation doesn't work well... I couldn't be able to test it

File: demos/dropdown/cyclow/es6/src/dropdown.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'in.init': () => state => ({goods, chosen: goods[0]}),
+    'dom.select': index => state => ({...state, chosen: goods[index]})
+  },
+  view: ({goods, chosen}) => ({
+    attrs: {id: 'app', class: chosen.type},
+    content: [
+      'I postit that ',
+      {
+        tag: 'select',
+        on: {change: (e, next) => next({select: e.selectedIndex})},
```
Comment (krawaller) [inline demos/dropdown/cyclow/es6/src/dropdown.js:16]: On debugging it seems that `e.selectedIndex` is undefined, the event object has no such property.

Which means that on line 8 the `index` is undefined, causing `goods[index]` to be undefined too, which finally causes `chosen` to be undefined on line 11.

Is Cyclow supposed to augment the change event on the dropdown with the `.selectedIndex` prop?

**Pull request merged**